src/redux/book/book.js

```diff
@@ -1,41 +1,57 @@
-// action
-import { REMOVE_BOOK, ADD_BOOK } from '../../util/constant';
-
-const initialState = [
-  {
-    title: 'book 1',
-    author: 'author 1',
-    id: 1,
-  },
-  {
-    title: 'book 2',
-    author: 'author 2',
-    id: 2,
-  },
-  {
-    title: 'book 3',
-    author: 'author 3',
-    id: 3,
-  },
-];
-
-// reducer
-
-export default (state = initialState, action) => {
+import { getData, removeData, uploadData } from '../../util/axios';
+// Actions
+const ADDBOOK = 'bookstore/books/ADDBOOKS';
+const DELETEBOOKS = 'bookstore/books/DELETEBOOKS';
+const READ_BOOKS = 'bookstore/books/READ_BOOKS';
+
+const initialState = [];
+
+// Reducer
+
+export function booksReducer(state = initialState, action) {
   switch (action.type) {
-    case ADD_BOOK:
-      return [...state, action.payload];
-    case REMOVE_BOOK:
-      return state.filter((i) => i.id !== action.payload.id);
-    default: return state;
+    case ADDBOOK: {
+      return [
+        ...state,
+        action.payload,
+      ];
+    }
+    case DELETEBOOKS: {
+      const ActualBooks = state.filter((book) => book.id !== action.payload);
+      return ActualBooks;
+    }
+    case READ_BOOKS: {
+      return action.payload;
+    }
+    default:
+      return initialState;
   }
+}
+
+// Action creators
+
+// export const addbook = (book) => ({ type: ADDBOOK, payload: book });
+export const addbook = (book) => async (dispatch) => {
```

Comment:
- [ ] Thunk action creator for fetching books should be created, kindly implement this it's a requirement